app/javascript/components/OrderManagerFilesetsForm.vue

```diff
@@ -159,11 +161,6 @@ export default {
   },
   watch: {
```

Comment:
I'm actually not sure that what I've offered here is the correct solution, as this assumed that the `unitLabel` will only *ever* be `f. ` or `p. ` - users may wish to offer a custom label.